src/locales/en.json

```
@@ -510,19 +510,16 @@
     "information": "Audio information"
   },
   "meta-search": {
-    "caption": "{openverse} does not currently index the sources listed above, but through this interface is offering convenient access to search services provided by other independent organizations. {openverse} has no control over the results that are returned. Do not assume that the results displayed in this search portal are under a CC license. Always verify that the work is actually under a CC license by following the link. If you are in doubt, you should contact the copyright holder directly, or try to contact the site where you found the content.",
+    "caption": "{openverse} has no control over the results, always verify that the work is actually under a CC license.",
```

```suggestion
    "caption": "{openverse} can not guarantee the accuracy of license information. Always verify that the work is actually under a CC license.",
```

"Openverse has no control" sounds aggressive to me. We can soften this language a little.